core/server/middleware/custom-redirects.js

```diff
@@ -1,5 +1,6 @@
 var fs = require('fs-extra'),
     _ = require('lodash'),
+    debug = require('debug')('ghost:custom-redirects'),
```

Comment:
Something to do one day but not high priority - this PR was just removing a bunch of things that were really annoying me whilst trying to build other things